packages/css/src/components/description-list/description-list.scss

```diff
@@ -0,0 +1,72 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+@import "../../common/breakpoint";
+@import "../../common/hyphenation";
+
+@mixin reset {
+  box-sizing: border-box;
+  list-style: none;
+  margin-block: 0;
+  margin-inline: 0;
+  -webkit-text-size-adjust: 100%;
+}
+
+.ams-description-list {
+  color: var(--ams-description-list-color);
+  display: grid;
+  font-family: var(--ams-description-list-font-family);
+  font-size: var(--ams-description-list-font-size);
+  font-weight: var(--ams-description-list-font-weight);
+  gap: var(--ams-description-list-gap);
+  grid-template-columns: 1fr;
+  line-height: var(--ams-description-list-line-height);
+  list-style-type: var(--ams-description-list-list-style-type);
+
+  &:not(.ams-description-list--rows) {
+    @media screen and (min-width: $ams-breakpoint-medium) {
+      grid-template-columns: 1fr 2fr;
+    }
+  }
+
+  @include reset;
+}
+
+.ams-description-list--inverse-color {
+  color: var(--ams-description-list-inverse-color);
+}
+
+.ams-description-list__row {
```

Comment:
Have you tried supporting multiple terms or descriptions without a row? We did need divs while we were floating and clearing things, but here’s an opportunity for named grid areas if I’m not overlooking something.

Reply:
Or just `grid-column-start: 2` on the details?

Reply:
Sure I've also come across documentation where it is allowed to have multiple terms and details per row. 

@alimpens  This is de case? It just happens to conflict if you also have multiple terms.

Reply:
I'm not sure I understand your question.

If we always set `grid-column-start: 1` on the terms and `grid-column-start: 2` on the details, we wouldn't need DescriptionListRow, right?

Reply:
That’s also what I was hoping we can do.

Reply:
Sure, I'm trying to come up with a solution for multiple terms though. I've [suggested](https://github.com/Amsterdam/design-system/pull/1139#discussion_r1535390371) to use an object instead of children. What do you guys think?

Reply:
Also see [this link](https://html.spec.whatwg.org/multipage/grouping-content.html#the-dl-element:~:text=The%20following%20example%20illustrates%20the%20use%20of%20the%20dl%20element%20to%20mark%20up%20metadata%20of%20sorts)

Reply:
Ah, I guess I didn’t quite understand what you meant there.

I’m not a fan of object props; they’re difficult to use and compose. Let’s keep using child components.

Having multiple terms is an edge case. We can happily put that out of scope.